cumulusci/tasks/tests/test_command.py

```diff
@@ -80,7 +81,10 @@ def test_handle_returncode(self):
         task_config = TaskConfig({"options": {"command": "ls"}})
         task = Command(self.project_config, task_config)
         with self.assertRaises(CommandException):
-            task._handle_returncode(1, "err")
+            with StringIO() as stderr:
+                stderr.write(u"err")
+                stderr.seek(0)
+                task._handle_returncode(1, stderr)
```

Comment:
The docs for subprocess say that `stderr` is "a bytes sequence, or a string if run() was called with an encoding, errors, or text=True." So I think we want to pass bytes for the test. I think you can just do `task._handle_returncode(1, io.BytesIO(b"err"))`

_handle_returncode will then need to decode the bytes